homeassistant/components/media_player/__init__.py

```diff
@@ -965,6 +962,7 @@ async def get(
             data, content_type = await player.async_get_media_image()
 
         if data is None:
+            _LOGGER.warning("Error retrieving proxied image from %s", image_url)
```

Comment:
@balloob do we want the warning level here? or did you intend for it to be debug still and i just misread the comment?

Reply:
Warning is good.